### PR TITLE
Vitaliy tolochniy labs

### PR DESCRIPTION
**Pull request by Vitaliy-Kovalevskiy**

### CODE REVIEW

File: labs/vitaliy_tolochniy/lab3/task1/1.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+""" 2. Визначити, чи є задане натуральне число досконалим, тобто рівним сумі всіх своїх дільників, крім
+самого цього числа (наприклад, число 6 досконале: 6 = 1 + 2 + 3). """
+
+a = float(input("Введите число: "))
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/lab3/task1/1.py]: Float value produces an infinite loop (just try 20.1 etc).
Use  "натуральне" for both user input prompt and input conversion like 
`int(input("Введите натуральное число: "))`

File: labs/vitaliy_tolochniy/lab3/task2/2.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+""" 6. Логічній змінній t присвоїти значення true або false залежно від того, є натуральне число k
+ступенем 3 чи ні. """
+
+k = float(input("Введите число: "))
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/lab3/task2/2.py]: Same to above

File: labs/vitaliy_tolochniy/lab3/task4/4.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+як дійсне число. """
+
+N = int(input("Введите число: "))
+if N <= 0:
+    print("Неверное число")
+else:
+    k = 1
+    product = 1
+    while k <= N:
+        product = product * k
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/lab3/task4/4.py]: Check `*=, +=` operators, not important

File: labs/vitaliy_tolochniy/lab4/task2/2.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+"""6. Дано рядок. Якщо в цьому рядку буква f зустрічається тільки один раз, виведіть її індекс. Якщо
+вона зустрічається два і більше разів, виведіть індекс її першого і останнього входження. Якщо
+буква f в цьому рядку не зустрічається, нічого не виводьте."""
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/lab4/task2/2.py]: `^M`?

File: labs/vitaliy_tolochniy/lab4/task5/5.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+поки не буде введено число 0. Видалити всі
+непарні числа даного списку та вивести його.'''
+
+list = []
+i = int(input('Вводите числа(для остановки введите 0): '))
+while i != 0:
+    list.append(i)
+    i = int(input('Вводите числа(для остановки введите 0): '))
+j = 0
+while j <= len(list)-1:
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/lab4/task5/5.py:11]: Please remind me to show some tricks with `.filter` and other methods, and list/dict comprehension

**Review by sen-den — CHANGES_REQUESTED** [review]
Great. Check comments please, a few fixes required.

File: labs/vitaliy_tolochniy/lab3/task1/1.py (hunk 1 above)
Comment (Vitaliy-Kovalevskiy) [reply labs/vitaliy_tolochniy/lab3/task1/1.py]: Okay

File: labs/vitaliy_tolochniy/lab3/task2/2.py (hunk 2 above)
Comment (Vitaliy-Kovalevskiy) [reply labs/vitaliy_tolochniy/lab3/task2/2.py]: Okay

File: labs/vitaliy_tolochniy/lab4/task5/5.py (hunk 5 above)
Comment (Vitaliy-Kovalevskiy) [reply labs/vitaliy_tolochniy/lab4/task5/5.py:11]: ok if I don't forget :)

File: labs/vitaliy_tolochniy/lab4/task2/2.py (hunk 4 above)
Comment (Vitaliy-Kovalevskiy) [reply labs/vitaliy_tolochniy/lab4/task2/2.py]: I don’t know what it is, the code doesn’t have it, but it gives an error

**Review by alexandrbarulin — APPROVED** [review]
ok

**Review by sen-den — APPROVED** [review]

**Review by alexandrbarulin — APPROVED** [review]

**Review by andxzrey — COMMENTED** [review]
ok

**Pull request merged**